### PR TITLE
ko-KR for upcoming update

### PR DESCRIPTION
**Pull request by OneKoin**
# SCP:SL Translation PR Template

## Language Name
ko-KR

## State Type of change (New language, changed existing Etc.)
translation updates for upcoming updates

# Checklist:
- [x] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation

### CODE REVIEW

File: Translations/ko/Class_Bio.txt [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+<color=#f00>SCP-096</color>은 구역에 존재하는 것만으로도 압박을 가하며, 분노하면 집단 전체를 신속하게 없앨 수 있습니다.
+<color=#f00>SCP-049-2</color>는 소생된 좀비로, 그 숫자가 늘어나면 게임의 흐름을 변화시킬 수 있습니다.
+위험으로부터 재단의 안전을 책임지는 엘리트 특무부대의 <color=#0096ff>부관</color>입니다. 주요 목표는 모든 <color=#f00>SCP</color>, <color=#008f1e>혼돈의 반란</color>과 <color=#FF8E00>D계급</color>을 제거하고 <color=#FFFF7CFF>과학자</color>를 안전하게 탈출시키는 것입니다.
+위험으로부터 재단의 안전을 책임지는 엘리트 특무부대의 <color=#003ecb>지휘관</color>입니다. 주요 목표는 모든 <color=#f00>SCP</color>, <color=#008f1e>혼돈의 반란</color>과 <color=#FF8E00>D계급</color>을 제거하고 <color=#FFFF7CFF>과학자</color>를 안전하게 탈출시키는 것입니다.
+위험으로부터 재단의 안전을 책임지는 엘리트 특무부대의 낮은 직급을 가진 <color=#63c3ff>사관생도</color>입니다. 주요 목표는 모든 <color=#f00>SCP</color>, <color=#008f1e>혼돈의 반란</color>과 <color=#FF8E00>D계급</color>을 제거하고 <color=#FFFF7CFF>과학자</color>를 안전하게 탈출시키는 것입니다.
+주어진 임무를 수행하여 목표를 달성하십시오.
+<color=#5b6371>시설 경비</color>의 목표는 <color=#FFFF7CFF>과학자</color>의 탈출을 돕고 곧 있을 <color=#f00>SCP</color> 의 위협을 멈추는 것입니다. <color=#0096FF>구미호 특무부대</color> 와 협력하여 방해가 되는 모든 것을 사살하십시오.
+<color=#f00>SCP-939</color>는 소리로 세상을 봅니다. 빠른 움직임과 조용한 발걸음을 가지고 있어 암살에 특화되어 있습니다.
+<color=#f00>SCP-939</color>는 소리로 세상을 봅니다. 빠른 움직임과 조용한 발걸음을 가지고 있어 암살에 특화되어 있습니다.
```
Comment (Tychus-Xiaoyi) [inline Translations/ko/Class_Bio.txt:18]: Thank you very much for your participation in the translation. But please do not add additional color coding

**Comment by OneKoin** [comment]
I changed those color code. There will be no additional color codes compared with default translation.

File: Translations/ko/Class_Bio.txt (hunk 1 above)
Comment (OneKoin) [reply Translations/ko/Class_Bio.txt]: > Thank you very much for your participation in the translation. But please do not add additional color coding

I changed those color code. There will be no additional color codes compared with default translation.

**Comment by Writrrr** [comment]
Those translations are pretty good. There is no need to say that this is about 10 times better than those default translation in version 9.1.3. You'd better check it out, and apply them. I dont know why this hadn't been approved for a month

**Comment by Rnen** [comment]
Please fix file conflicts (pull the newest version into yours). Conflicting file: Credits.txt

Also @Tychus-Xiaoyi please recheck requested changes.

**Comment by OneKoin** [comment]
> Please fix file conflicts (pull the newest version into yours). Conflicting file: Credits.txt
> 
> Also @Tychus-Xiaoyi please recheck requested changes.

Should I add my nickname or just pull the newest version without editing?

**Comment by Rnen** [comment]
Uhhh...
image

**Comment by OneKoin** [comment]
> Uhhh...
> image
> 

That's not a problem as MP2 translation have been merged. I'll recreate those translations by tommorow.

**Comment by Dat-Joseph** [comment]
I close this pull request as you will most likely make a new one

**Pull request closed without merging**